### PR TITLE
Don't miss out - show indication of upcoming deadlines [completes #83]

### PR DESCRIPTION
**Pull request by sh0oki**

### CODE REVIEW

**Comment by sh0oki** [comment]
@rootux I'm still waiting for the list of relevant days from Eitan. Please don't merge yet :)

**Comment by sh0oki** [comment]
image

**Comment by sh0oki** [comment]
Resolves #83

File: app/controllers/camps_controller.rb [hunk 1]
```diff
@@ -31,7 +31,6 @@ def new
   end
 
   def edit
-    @camp = Camp.find params[:id]
```
Comment (rootux) [inline app/controllers/camps_controller.rb:34]: What is this? not needed?

File: config/lockdown.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+development:
```
Comment (rootux) [inline config/lockdown.yml]: So we will need also `production` one?

File: app/controllers/camps_controller.rb (hunk 1 above)
Comment (sh0oki) [reply app/controllers/camps_controller.rb:34]: Nope, `load_camp!` takes care of it

**Comment by rootux** [comment]
@sh0oki Can you also add it when a user pressed on 'Save dream and Finish' - flash this message?

File: config/lockdown.yml (hunk 2 above)
Comment (sh0oki) [reply config/lockdown.yml]: oops! Good point :) adding now

**Comment by sh0oki** [comment]
@rootux Sure. 
Only after save? Maybe we can just add it in the 'show' page for anybody logged in that is looking at his own dream?

**Comment by rootux** [comment]
Hmmm… sure let’s do that - but make sure that only if any date is before. otherwise don’t display those - so after the dates are passed they won't be bothered with those messages.
I mean there could be another message - like 10 days left till ending of distribution of coins. as long as there is a message left - show it. otherwise if passed - hide it.

**Comment by sh0oki** [comment]
Of course :) That's what Lockdown is supposed to do. We'll also be able to use `Lockdown.allowed?(action)` to determine if an action is still allowed.

**Pull request merged**